<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="calculate_volume">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate volume</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selection_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Threshold distance for 90 deg cavity</string>
   </property>
   <widget class="QRadioButton" name="dist_90_auto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto (X times window size):</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="dist_90_manual">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual distance (Å):</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dist_90_auto_value">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dist_90_manual_value">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>40.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Cavity volume selection</string>
   </property>
   <widget class="QRadioButton" name="volume_largest_check">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Largest</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="volume_close_to_center_check">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Closest to center</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="volume_all_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Cavity properties</string>
   </property>
   <widget class="QCheckBox" name="hydro_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hydrophobicity</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="aro_check">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Aromatic contacts</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="sas_check">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SASA</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="esp_check">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ESP</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Hydrophobicity method</string>
    </property>
    <widget class="QComboBox" name="hydro_method">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Ghose</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Crippen</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="hydro_dist">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Fauchere</string>
     </property>
     <item>
      <property name="text">
       <string>Fauchere</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fauchere2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Audry</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OnlyValues</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dist. func.:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>102</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Metal charge for ESP</string>
    </property>
    <widget class="QSpinBox" name="metal_charge_text">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>45</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>23</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Metal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>22</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Charge</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="metal_name_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Pd</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="metalESPcheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>21</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>102</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Charge Model for ESP</string>
    </property>
    <widget class="QComboBox" name="chargeModelComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>eem</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mmff94</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gasteiger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>qeq</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>qtpie</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>eem2015ha</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>eem2015hm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>eem2015hn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>eem2015ba</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>eem2015bm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>eem2015bn</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>38</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid size (Å):</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="grid_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
